superset/utils/core.py

```diff
@@ -1669,3 +1669,10 @@ def get_time_filter_status(  # pylint: disable=too-many-branches
             )
 
     return applied, rejected
+
+
+def format_list(items: Sequence[str], sep: str = ", ", quote: str = '"') -> str:
+    items = [
+        "".join((quote, item.replace(quote, "\\" + quote), quote)) for item in items
```

Comment:
Nit: can probably use a formatted string instead of join here to make it easier to read.

```python
def format_list(items: Sequence[str], sep: str = ", ", quote: str = '"') -> str:
    return sep.join(f'{quote}{x.replace(quote, "\\" + quote)}{quote}' for x in items)
```

Reply:
That was my original approach, but then I discovered that you cannot use backslashes inside an f-string.

Reply:
```python
def format_list(items: Sequence[str], sep: str = ", ", quote: str = '"') -> str:
    quote_escaped =  "\\" + quote
    return sep.join(f'{quote}{x.replace(quote, quote_escaped)}{quote}' for x in items)
```

How about this?

Reply:
Cool, let me change it.